### ISSUE TITLE
!] error: Could not find a valid passphrase.

### ISSUE THREAD

**Issue by wooilok123** [RickdeJager/stegseek#8]
I ran this command and got an error which says "Could not find a valid passphrase."

stegseek brooklyn99.jpg rockyou.txt

[!] error: Could not find a valid passphrase.

![image](https://user-images.githubusercontent.com/68337442/128387133-0b5607e9-2e61-44e5-81e2-7f06fc2d30b3.png)

**Comment by RickdeJager** [comment]
Can you provide some more information?

You didn't post the file or password so I'm kinda in the dark here, but I'm guessing you're playing that one THM room? In that case, stegseek should definitely work. For example, here's someone else's writeup with the same wordlist and exact same release of stegseek:
https://chaudhary1337.github.io/p/tryhackme-brooklyn-nine-nine-writeup-all-paths/
  
Some trouble shooting steps:
* Make sure you're using the _exact_ source file. Compressing or taking a screenshot of a steghide-stego file breaks the contents.
* Make sure your wordlist file is confirmed good. The sha256sum should match:
```
[/tmp/tmp.GVsiXeNNft]~$ sha256sum rockyou.txt 
6dfa76aa0e02303994fd1062d0ac983f0b69ece5474d85a5bba36362e19c1076  rockyou.txt
```

If those don't help, try running stegseek in single threaded mode just in case.
```
stegseek brooklyn99.jpg rockyou.txt -t 1 -v
```
(if it turns out to be a threading issue, please report back so I can fix it.)

**Comment by wooilok123** [comment]
Thanks for the reply, RickdeJager.

You're right, I realized that the rockyou.txt file I have is corrupted and sha256sum doesn't match what you shown above. I changed the file and everything works fine and smooth. 
Great tool, managed to obtain the passphrase from rockyou file in just a second! 
![image](https://user-images.githubusercontent.com/68337442/128482377-50cf3aa3-82aa-41c8-be5a-c066a3a4d436.png)
